nuxeo-webdav-test/src/test/java/org/nuxeo/ecm/webdav/WebDavClientTest.java

```diff
@@ -202,6 +202,33 @@ public void testListFolderContents() throws Exception {
         }
     }
 
+    @Test
+    public void testListFolderContentsSpecialName() throws Exception {
+        String folderName = "foo bar[1] caf\u00e9";
+        DocumentModel folder = session.createDocumentModel("/workspaces/workspace", folderName, "Folder");
+        session.createDocument(folder);
+        session.save();
```

Comment:
Still useful since committing the transaction right ater?

Reply:
I know the transaction does a `save()` but I prefer to be explicit. Also if there's an error during the save triggered by the commit it's sometimes harder to get a proper stack trace.